### DISCUSSION TITLE
Capacidad Maxima de cola de espera

### DISCUSSION THREAD

**Discussion by aangel-sama** [IIC2133-PUC/2024-2#104] [Tarea 1] [answered]
La cola no tiene capacidad maxima cierto? En ese caso, podemos nosotros definir una capacidad maxima, y si se supera esa capacidad maxima, redimensionar la capacidad maxima de la cola. O debemos usar otro metodo?

**Comment by tomastrivino** [comment, marked as answer]
Puedes modelar la cola de espera como lo estimes conveniente, lo importante es que te permita cumplir con las complejidades y el flujo de los eventos respectivos. Existen diversas estructuras de datos que podrías usar (algunas con o sin "límite", dependiendo de qué es lo que entendamos como límite).
Saludos
